### PR TITLE
Update Github Actions

### PR DESCRIPTION
**Pull request by sanatd33**
fix github actions by changing the ros from foxy to humble

### CODE REVIEW

**Review by jacksherling — APPROVED** [review]
looks good to me!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
   build-and-test:
     runs-on: ubuntu-latest
     container:
-      image: robojackets/robocup-software:foxy
+      image: robojackets/robocup-software:humble
```
Comment (sid-parikh) [inline .github/workflows/ci.yml:10]: did we push a humble tag of this image?

Comment (sid-parikh) [reply .github/workflows/ci.yml:10]: i see that we did. it says on docker hub that I did actually, LOL. how did you guys manage that?

**Review by sid-parikh — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (sanatd33) [reply .github/workflows/ci.yml:10]: Yup I made a humble tag on Sundays meeting

**Comment by sid-parikh** [comment]
@sanatd33 you should merge this and then rebase every branch with an open PR so the new checks run
(or make everyone else do it of course)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (sanatd33) [reply .github/workflows/ci.yml:10]: You were still signed in on the shop computer so I logged in on your account

**Pull request merged**